dist/index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+	
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+	<section class="container">		
+			<h1>Leaderboard</h1>
+		<div id="wrapper">
+			<div class="show-scores">
+				<div class="recent-scores">
+					<h2>Recent scores</h2>
+                    <button class="refresh" type="button">Refresh</button>
+				</div>
+				<ul class="scores">
+                   
+				</ul>
+				
+			</div>
+			<div class="add-scores">
+				<div class="add-heading">
+					<h2>Add your score</h2>
+				</div>
+				<form class="add-input">
+					<label for="name"></label>
+					<input required type="text" id="name" name="name" placeholder="Your name">
+					<label for="score"></label>
+					<input required type="number" id="score" name="score" placeholder="Your score" >
+					<button type="submit" class="submit-button">Submit</button>
+				</form>
+			</div>
+		</div>
```

Comment:
- [ ] You need to follow the wireframe design to style your application. Please work on the design and make sure the application matches the wireframe design.

### Your design

![image](https://user-images.githubusercontent.com/83803414/186185678-f83959e0-fd90-43af-aee9-4b404898c7e0.png)

### Wireframe design

![image](https://user-images.githubusercontent.com/83803414/186185726-6c0c153e-7e5c-48ef-ad11-30a15240f7a3.png)